flipperdevices/flipperzero-good-faps
# Picopass read card using nr-mac

# What's new

- Can save partial card when unknown key
- Can emulate partial card and capture nr-mac (named for csn_epurse)
- Can check for csn_epurse file and attempt to use nr-mac for auth

# Verification 

- Read iClass SE card (or other card without known key)
- Save partial
- Emulate partial
- Present emulation to read
- Feel haptic, see "NR-MAC Saved!" popup
- Try card read again
- See "SE enabled" and "SIO" + menu option
- Select Menu
- See "Save" (but not "save as lfrfid", etc)

# Checklist (For Reviewer)
